core/inputs/input.ts

```diff
@@ -298,6 +293,10 @@ export class Input {
       this.connection.dispose();
     }
   }
+
+  protected makeConnection(type: number): Connection {
```

Comment:
Just curious, why isn't the type of `type` `ConnectionType`?

Reply:
upon rereading this sentence: @_@

Reply:
Fixed!

On the block version of this method we used to pass `inputTypes` to it sometimes, which isn't compatible with `ConnectionType`, so we had to use `number` instead. But since the code has been reorganized, that is no longer an issue =) Everything has been switched to use `ConnectionType`!